BUILD.md

```diff
@@ -72,31 +77,69 @@ below, built in LLVM, and Xcode LLVM don't support the C++20 feature set require
 API isn't support on macOS, Vulkan must be used. Additionally Vulkan must be used through the 
 Molten-VK compatibility layer
 
-### On Apple Silicon Macs, Rosetta 2 and the x86_64 version of Homebrew must be used
-
-You can skip this section if you have an Intel Mac. Every time you compile, you need to perform steps 2.
-
-1. `softwareupdate --install-rosetta` # Install Rosetta 2 if you don't have it. This only has to be done once
-2. `arch -x86_64 zsh` # run an x64 shell
-
-### Installing brew
-
-1. `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
-2. `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` # set x86_64 brew env
-
-### Installing dependencies
-
-`brew install boost git cmake llvm ninja nasm molten-vk`
-
 ### Build Cemu using cmake and clang
-1. `git clone --recursive https://github.com/cemu-project/Cemu`
-2. `cd Cemu`
-3. `cmake -S . -B build -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/local/opt/llvm/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm/bin/clang++ -G Ninja`
-4. `cmake --build build`
-5. You should now have a Cemu executable file in the /bin folder, which you can run using `./bin/Cemu_release`.
 
-#### Troubleshooting steps
-- If step 3 gives you an error about not being able to find ninja, try appending `-DCMAKE_MAKE_PROGRAM=/usr/local/bin/ninja` to the command and running it again.
+Not all steps apply all the time.
+* Certain steps are only necessary if you have an Apple Silicon Mac. Steps that are necessary for an Apple Silicon Mac are marked with a :unicorn: emoji, and steps that are necessary for an Intel Mac are marked with a :ringed_planet: emoji.
+  * :warning: Note that there are steps that aren't marked for Intel Macs, and **Intel Mac users should skip these steps**.
+* Certain steps are only need to be done once. Steps that only need to be done once are marked with a :mortar_board:, and steps that need to be done every time are marked with a :repeat: emoji.
+* Certain steps are sometimes optional. These steps are marked with the :soap: emoji. See the accompanying note to understand when they apply.
+
+1. :mortar_board: :unicorn: Install Rosetta 2.
+   ```
+   softwareupdate --install-rosetta
+   ```
```

Comment:
Same as before, and code blocks too should be separated by blanks, see [MD031](https://github.com/DavidAnson/markdownlint/blob/main/doc/md031.md) (applies to more or less the whole section). Also, [fenced code blocks should specify the file type](https://github.com/DavidAnson/markdownlint/blob/main/doc/md040.md) ([`text` or `plaintext`](https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md) is ok for plain text - code blocks showing commands to put into a terminal shouldn't generally use the `sh`, `shell`, or `bash` type, as that's specifically for shell scripts).